### PR TITLE
fix load test shell

### PR DESCRIPTION
**Pull request by safeie**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,4 +20,4 @@ __debug_bin
 
 main
 dist/
-
+cmd/zinc
```
Comment (prabhatsharma) [inline .gitignore:23]: What exactly would this achieve? Would this not lead to ignoring main.go ?

Comment (safeie) [reply .gitignore:23]: the binary zinc should not commit to git, so ignore it.

```
cd zinc/cmd
go build
```
will generate binary zinc, so i ignore: `cmd/zinc`

**Comment by safeie** [comment]
fix some panic when index check

**Comment by prabhatsharma** [comment]
LGTM

**Pull request merged**